include/deal.II/grid/tria_description.h

```diff
@@ -448,27 +448,27 @@ namespace TriangulationDescription
 
 
     /**
-     * Construct a construction::Description. In contrast
+     * Construct a TriangulationDescription::Description. In contrast
      * to the function above, this function is also responsible for creating
      * a serial triangulation and for its partitioning (by calling the
-     * provided std::functions). Internally only selected processes (every
-     * n-th/each root of a group of size group_size) create a serial
+     * provided `std::functions' objects). Internally only selected processes (
```

Comment:
```suggestion
     * provided `std::function` objects). Internally only selected processes
```
?
And move the final brace to the next line.

Reply:
@kronbichler Done.